Header header
# plane coefficients
float32[4] coefficients
# cells
geometry_msgs/Point[] cells
